Tests/test_AlignInfo.py

```diff
@@ -139,7 +139,7 @@ def test_pseudo_count(self):
         ic = summary.information_content(e_freq_table=expected,
                                          log_base=math.exp(1),
                                          pseudo_count=1)
-        ic_vector = summary.ic_vector.values()
+        ic_vector = list(summary.ic_vector.values())
```

Comment:
Why is the list needed here? More importantly, how do we guarantee the order of the list?